### PR TITLE
feat: :sparkles: Added Tailwind functionality.

### PR DESCRIPTION
**Pull request by kphutiyagae**
* Installed Tailwindcss
* initialized tailwind.config and added properties to config.
* Added tailwind and tailwind-watch scripts to trans-pile utility classes to css.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -23,6 +25,7 @@
     "eslint-config-prettier": "^8.7.0",
     "eslint-plugin-react": "^7.32.2",
```
Comment (MihlaliMgwebi) [inline package.json]: Add rules for your linter or remove the script if your feature doesn't use this.

File: tailwind.config.cjs [hunk 2]
```diff
@@ -0,0 +1,8 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: ["./src/**/*.{html,js,ts,tsx}"],
```
Comment (MihlaliMgwebi) [inline tailwind.config.cjs]: Remove js since you won't use it in this project.

File: src/stylesheets/index.css [hunk 3]
```diff
@@ -0,0 +1,14 @@
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+html,
+body {
```
Comment (MihlaliMgwebi) [inline src/stylesheets/index.css:6]: Like this CSS reset,

**Review by MihlaliMgwebi — CHANGES_REQUESTED** [review]
Few comments. Will approve after changes.

File: package.json (hunk 1 above)
Comment (kphutiyagae) [reply package.json]: Done.

File: tailwind.config.cjs (hunk 2 above)
Comment (kphutiyagae) [reply tailwind.config.cjs]: Noted, removed.

File: src/stylesheets/index.css (hunk 3 above)
Comment (kphutiyagae) [reply src/stylesheets/index.css:6]: Thank you

**Review by MihlaliMgwebi — APPROVED** [review]

**Pull request merged**